### DISCUSSION TITLE
Support for streaming media

### DISCUSSION THREAD

**Discussion by CorneliusZee** [deepgram/community#397] [Feature Requests]
I currently cannot find any support for streaming media manifest files, Eg:

- m3u8
- f4m
- ismc
- mpd

file extensions.

**Comment by jjmaldonis** [comment]
Hi @CorneliusZee, we support the vast majority of audio data types. Have you tried running these data types through our streaming API? If the audio is transcribed, it works. It is doesn't work, you can use a tool like ffmpeg to convert the data type. Some of these data types are very rare. What are you looking to do?

**Comment by alchemician** [comment]
Need support for m3u8 stream
